doc/telescope.txt

```diff
@@ -592,6 +592,9 @@ builtin.git_branches({opts})                          *builtin.git_branches()*
 
     Parameters: ~
         {opts} (table)  options to pass to the picker
+        
+    Fields: ~
```

Comment:
Typically, we add docs via EmmyLua annotations which are then converted into docs via https://github.com/tjdevries/tree-sitter-lua through CI. 

Here, you can fix it by just adding the identical annotation analogously to `lua/telescope/builtin/init.lua` right below L190 as per other annotations in the file with `@field` argument.

Reply:
I think I got it.